### DISCUSSION TITLE
Libellé Formulaires en ligne dans l'entête

### DISCUSSION THREAD

**Discussion by maxpouliot** [MTESSDev/FRW#138] [Annonces]
Est-ce possible via le fichier de config de modifier / enlever le libellé formulaires en ligne qui est dans l'entête? 

Merci!

**Comment by LarcM7** [comment]
Bonjour,

Oui c'est possible de le modifier, dans la config "default" au niveau du système autorisé ou au niveau du formulaire (principe du cône). Je ne sais pas s'il est possible de l'enlever complètement. Mais ce n'est pas recommandé, il faut y mettre quelque chose pour identifier votre organisme.

https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=personnalisationBase

![nomSiteMOLigne1](https://github.com/MTESSDev/FRW/assets/129791924/efcf9cf5-3583-474a-a87a-d1f2daf009fd)

N'hésitez pas s'il y a quoi que ce soit.
